extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -505,7 +511,8 @@ export abstract class AzureAuth implements vscode.Disposable {
 		const account = {
 			key: {
 				providerId: this.metadata.id,
-				accountId: key
+				accountId: key,
+				accountVersion: AzureAuth.ACCOUNT_VERSION,
```

Comment:
wait, key has itself in the field?

Reply:
yeah there's technically the accountKey which contains all the information about the provider, and then the accountId is also technically a key